Api refrence update tasks and keys (#1796)
meilisearch/documentation: reference/api/error_codes.md

### `invalid_task_status`

> The requested task status is invalid. Please use one of [these four possible values](/reference/api/tasks.md#status).

Reviewer: Future-proofing: if you don't use an exact number you won't have to change it later when possible values are added or removed.

```suggestion
The requested task status is invalid. Please use one of the [possible values](/reference/api/tasks.md#status).
```

### `invalid_task_type`

> The requested task type is invalid. Please use one of the values defined in our [asynchronous operations guide](/reference/api/tasks.md#type).

Reviewer: ```suggestion
The requested task type is invalid. Please use one of the [possible values](/reference/api/tasks.md#type).
```
